MbDotNet.Acceptance.Tests/AcceptanceTests/CanCreateImposterTest.cs

```diff
@@ -0,0 +1,56 @@
+using System.Net;
+using FluentAssertions;
+using MbDotNet.Exceptions;
+
+namespace MbDotNet.Acceptance.Tests.AcceptanceTests
+{
+    internal class CanCreateImposterTest
+    {
+        private readonly MountebankClient _client;
+        private int imposterPort = 6000;
+
+        public CanCreateImposterTest(MountebankClient client)
+        {
+            _client = client;
+            
+        }
+
+        public void Run()
+        {
+            DeleteAllImposters();
+            CreateImposter();
+            VerifyImposterHasBeenCreated();
+            DeleteAllImposters();
+        }
+
+        private void DeleteAllImposters()
+        {
+            _client.DeleteAllImposters();
+        }
+
+        private void VerifyImposterHasBeenCreated()
```

Comment:
You mentioned that you plan to add a GetImposters method in a future PR. Make sure to update these tests to utilize that for verification once that is added so we no longer need to rely on the exceptions.